app/components/moderation-list-row/component.js

```diff
@@ -11,7 +32,25 @@ export default Ember.Component.extend({
         rejected: 'fa-times-circle-o rejected'
     },
 
+    gtDay: Ember.computed('submission.dateLastTransitioned', function() {
+        return moment().diff(this.get('submission.dateLastTransitioned'), 'days') > 1;
+    }),
+
+    relevantDate: Ember.computed('submission.dateLastTransitioned', 'gtDay', function() {
+        return this.get('gtDay') ?
+            moment(this.get('submission.dateLastTransitioned')).format('MMMM DD, YYYY') :
+            moment(this.get('submission.dateLastTransitioned')).fromNow();
```

Comment:
I like this change, but concerned on how the senetece generated by fromNow() (e.g. 44 seconds ago) will be translated.

Reply:
also, do we need to check with product on wording changes

Reply:
Oh good point. I hadn't considered that.

Reply:
Okay, [moment supports internationalization](https://momentjs.com/docs/#/i18n/) so if there are multiple languages I think we would just need to set the language in https://github.com/CenterForOpenScience/ember-osf-reviews/blob/develop/app/routes/application.js.

Reply:
I'll double check on the wording changes. We couldn't use what was in the mockups due to the difficulty of getting the moderator's name.

Reply:
Wording approved by product and MOD-115 added to backlog to update the wording with the moderator's name.